go.mod

```diff
@@ -3,155 +3,193 @@ module github.com/kava-labs/kava
 go 1.18
 
 require (
-	github.com/cosmos/cosmos-proto v1.0.0-alpha6
-	github.com/cosmos/cosmos-sdk v0.45.10
-	github.com/cosmos/ibc-go/v3 v3.4.0
-	github.com/ethereum/go-ethereum v1.10.16
+	cosmossdk.io/math v1.0.0-beta.6.0.20230216172121-959ce49135e4
```

Comment:
Is it possible to use `github.com/cosmos/cosmos-sdk` instead of `cosmossdk.io/math` set in their `go.mod`?

I get this error when trying to use `github.com/cosmos/cosmos-sdk/math`

```text
go: downloading github.com/cosmos/cosmos-sdk/math v1.0.0-rc.0
go: github.com/cosmos/cosmos-sdk/math@v1.0.0-rc.0: parsing go.mod:
	module declares its path as: cosmossdk.io/math
	        but was required as: github.com/cosmos/cosmos-sdk/math
```

Reply:
ah, yup. ok that makes sense. easiest to just keep it as is then because they define that as their module path in go.mod.

safe to disregard, but worth the mental note.